### ISSUE TITLE
Configure timeout of sync request

### ISSUE THREAD

**Issue by hicksy** [phonegap/phonegap-plugin-contentsync#23]
Is it possible to increase the timeout period? 

Found on poor connection I'm getting an error timeout after ~ 10 seconds (60% way through zip download)

**Comment by timkim** [comment]
We can definitely do that. Which platform has the quick timeout? I think they are a little different on all the platforms.

**Comment by hicksy** [comment]
Thanks, it will be a useful addition. 

I was working with the iOS platform at the time.  (was only running in Simulator if that matters?)

**Comment by mwbrooks** [comment]
Ideally, we should standardize the timeout across all platforms (constant
variable) and allow a sync task to customize the timeout for their own
needs (using the options object).

**Comment by imhotep** [comment]
yeah I agree it should be customizable with a reasonable default on all platforms. Currently it's 15 seconds on iOS and Android I believe.
